README.md

```diff
@@ -316,28 +317,26 @@ We have a few known features we want to take on from here while we remain open f
 If you would like to propose a feature, please [open an Issue](https://github.com/intelsdi-x/snap/issues)) that includes RFC in it (for [request for comments](https://en.wikipedia.org/wiki/Request_for_Comments)).
 
 ## Community Support
-This repository is one of **many** projects in the **Snap framework**. Discuss your questions about Snap by reaching out to us:
-
-* Through GitHub Issues. Issues is our home for **all** needs: Q&A on everything - installation, request for events, integrations, bug issues, futures. [Open up an Issue](https://github.com/intelsdi-x/snap/issues) and know there's no wrong question for us.
-* We also have a [Slack team](https://intelsdi-x.herokuapp.com/) where we hang out
-* Submit a blog post on your use of Snap to [our Medium.com publication](https://medium.com/intel-sdi)
+This repository is one of many in the Snap framework and [has maintainers supporting it](docs/MAINTAINERS.md). We love contribution from our community along the way. No improvement is too small.
 
-The full project lives here, at http://github.com/intelsdi-x/snap.
+This note is especially important for plugins. While the Snap framework is hardened through tons of use, **plugins mature at their own pace**. If you have subject matter expertise related to a plugin, please share your feedback on that repository. 
 
 ## Contributing
-We encourage contributions from the community. No improvement is too small. Snap needs:
+We encourage contributions from the community. Snap needs:
 
-* _Feedback_: try it and tell us about it through issues, blog posts or Twitter
-* _Contributors_: We can always use more eyes on the core framework and its testing
+* _Contributors_: We always appreciate more eyes on the core framework and plugins
+* _Feedback_: try it and tell us about it on [our Slack team](https://intelsdi-x.herokuapp.com/), through a blog posts or Twitter with #SnapTelemetry
```

Comment:
Change 'our Slack team' to just 'Slack' or reword for clarity. Here are we offering users to post a blog to our Medium.com publication? This might be a good place for that link.